rest-service/manager_rest/rest/resources_v3_1/labels.py

```diff
@@ -10,12 +10,29 @@ class DeploymentsLabels(SecuredResource):
     @authorize('labels_list')
     def get(self):
         """Get all deployments labels' keys"""
-        current_tenant = utils.current_tenant._get_current_object()
+        current_tenant = utils.current_tenant
         results = (db.session.query(models.DeploymentLabel.key)
                    .join(models.Deployment)
                    .filter(models.Deployment._tenant_id == current_tenant.id)
                    .distinct()
                    .all())
 
         keys_list = [result.key for result in results]
-        return {'metadata': {}, 'items': keys_list}
+        return {'metadata': {'total': len(keys_list)}, 'items': keys_list}
```

Comment:
Are we not using paging on this? I think the paging stuff has logic for injecting total, etc already?

Reply:
I frankly don't know. @tehasdf ?

Reply:
of course we're not using pagination, that is only added by the storage-manager, in .list(), and this is a bare db.session.query

Reply:
Then do we need to add 'total', since it'll be added later?